taxi/views.py

```diff
@@ -59,6 +80,20 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
     queryset = Car.objects.select_related("manufacturer")
```

Comment:
Do you need to define queryset in the class when you override the get_queryset method?